geps/gep-3875/index.md

```diff
@@ -0,0 +1,192 @@
+# GEP-3875: BackendTLS consumer overrides and context awareness
+
+* Issue: [#3875](https://github.com/kubernetes-sigs/gateway-api/issues/3875)
+* Status: Provisional
```

Comment:
If we want to get this into v1.4, I think this should go straight to implementable.
```suggestion
* Status: Implementable
```